src/Data.h

```diff
@@ -73,6 +74,8 @@ struct DataFrame {
   float      HEADING_GPS                     =                                0;
   SunsetPredictor::GPSTime GPS_TIME;
   bool SWITCH_TO_MANUAL = false;
+  float      ALTITUDE_OFFSET                 =                                0;
+  float      ALTITUDE_OFFSET_GAIN            =                             4e-5;
```

Comment:
Is there a nice interpretation of this gain as a frequency cutoff of a first order filter? (But tbh I'm also fine with "nah it just looks good like this.")

Reply:
yeah. Since it's first order, `time_constant = 1/(corner_freq)`, and the time contstant is like the time it takes the step response to get to `1-1/e` which is about .63. And u can find the value of the step response of the filter after 30 minutes with `1-(1-gian)^(30*60*20)` and if ur lazy like me rather than solve for the gain u plug that shit into google and fiddle the gain until the result is about .63